src/graph_world/beam/generator_config_sampler.py

```diff
@@ -88,6 +88,18 @@ def SampleConfig(self, marginal=False):
             param_value = spec.default_val
       # If the param val is still None, give it a random value.
       if param_value is None:
-        param_value = spec.sampler_fn(spec)
+        # Allow parameters not being sampled to remain in output as null
+        if spec.sampler_fn is None: 
+          param_value = None
+        else: 
+          param_value = spec.sampler_fn(spec)
       config[param_name] = param_value
-    return config, marginal_param, fixed_params
\ No newline at end of file
+    return config, marginal_param, fixed_params
+
+  def _AddParamSamplerSpecs(self, extra_param_sampler_specs, overwrite=False):
```

Comment:
Paired with my suggestion on `src/graph_world/nodeclassification/generator_wrappers.py`, this function can be removed.